Neos.Neos/Classes/Controller/Module/Administration/SitesController.php

```diff
@@ -217,23 +218,42 @@ public function newSiteAction(Site $site = null)
     {
         $sitePackages = $this->packageManager->getFilteredPackages('available', null, 'neos-site');
         $documentNodeTypes = $this->nodeTypeManager->getSubNodeTypes('Neos.Neos:Document', false);
+
+        $generatorServiceIsAvailable = $this->packageManager->isPackageAvailable('Neos.SiteKickstarter');
+
+        $generatorServices = [];
+
+        if ($generatorServiceIsAvailable) {
+            $generatorClasses = $this->reflectionService->getAllSubClassNamesForClass(AbstractSitePackageGenerator::class);
```

Comment:
IMHO the better way is to define a `SiteGeneratorInterface` and select classes implementing this interface